mlflow/langchain/__init__.py

```diff
@@ -800,10 +801,16 @@ def load_model(model_uri, dst_path=None):
     return _load_model_from_local_fs(local_model_path)
 
 
-def _load_code_model(code_path: Optional[str] = None):
+@contextmanager
+def _config_path_context(code_path: Optional[str] = None):
     _set_config_path(code_path)
+    yield
+    _set_config_path(None)
```

Comment:
Is it not necessary to reset this state to the state that existed prior to entrance to this local context manager? Would a finally block to reset be better than assignment to `None` ?